package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "country-details",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "https://restcountries.com/v3.1/",
+  "homepage": "http://localhost:3000/",
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.3.5",
+    "http-proxy-middleware": "^2.0.6",
+    "localforage": "^1.10.0",
+    "match-sorter": "^6.3.1",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.10.0",
+    "react-scripts": "5.0.1",
+    "sort-by": "^1.2.0",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
- [ ] [REQUIRED] Good job so far! When I start the project, the project starts, but the data in the `API` schema is not fully displayed, and as a result, the project display `loading`. Please fix it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/233705886-be7123da-282e-4212-b717-db0484beda14.png)